src/components/BuySellForm/BuySellAmount/BuySellAmount.scss

```diff
@@ -1,4 +1,19 @@
 .buy-sell-amount {
+    &__amount {
+        margin-bottom: 0.2rem;
+
+        &--disabled {
+            border-color: #eaeced;
+
+            & .deriv-input__field {
+                color: #999999;
+            }
+
+            &:hover {
+                border-color: #eaeced !important;
+            }
+        }
```

Comment:
i couldn't override the input on hover effect when we disable the field to match prod without !Important. I tried directly accessing the input that also didnt work

Reply:
can we pass `disabled` as variant?

Reply:
![Screenshot 2024-08-14 at 1 21 49 PM](https://github.com/user-attachments/assets/4fbd3de4-3398-4fa5-854b-d9d1e8eb5a09)

I've done that already, but it doesn't disable the hover styles 😓 nor does it change the colour of the input text